tox.ini

```diff
@@ -8,19 +8,30 @@ commands =
 deps =
     zope.testrunner
 
-[testenv:lint]
-skip_install = true
-commands =
+;[testenv:format]
+;skip_install = true
+;commands =
 ;    flake8 repoze tests setup.py
 ;    isort --check-only --df repoze tests setup.py
 ;    black --check --diff repoze tests setup.py
-    python setup.py sdist
-    twine check dist/*
-    check-manifest
-deps =
+;deps =
 ;    flake8
 ;    black
 ;    isort
-    twine
-    readme_renderer
+
+[testenv:build]
+skip_install = true
+commands =
+    # clean up build/ and dist/ folders
+    python -c 'import shutil; shutil.rmtree("dist", ignore_errors=True)'
+    python setup.py clean --all
```

Comment:
No longer call setup.py directly, use the new `build` package instead (replaces the pep517.build)

This also does away with he sdist call/pip wheel call.

Reply:
Thanks, @bertjwregeer! Updated in 1663057. Please let me know.